.envrc.global

```diff
@@ -6,5 +6,6 @@
 # An https URL must be used for the repo, or docker won't be able to clone it.
 export OTP_REPO=https://github.com/opentripplanner/OpenTripPlanner.git
 # The commit can be either a hash or a branch, but only hashes should be used in prod.
-# This is somewhere after 2.4.0 but before 2.5.0, in order to use the numberOfTransfers field on Itinerary
-export OTP_COMMIT=5da22683573f94b5d32f4711e29ff2252c28fcac
+
+# This is somewhat after 2.5.0
+export OTP_COMMIT=69dca4ee82fc7b8fd96c610b176e40d750007841
```

Comment:
question: is there a reason we can't use the commit for the 2.5.0 release? According to https://github.com/opentripplanner/OpenTripPlanner/releases/tag/v2.5.0 the relevant SHA is b301ea7c70f2ce7bb22c7f8e7fba6f8654cea3a5

Reply:
To that point, why not the latest `dev-2.x` instead? (The OTP devs have assured me that releases are arbitrary and anything on `dev-2.x` is just as stable, but I'm not sure we want to take that risk.)

Reply:
To Emma's note, I'd rather not use `dev-2.x` since that's a moving target and, while stable, will totally contain breaking changes.

To Paul's note, I'll admit sometimes I specify non-major-version commits here to let us make use of a particular change (ex. major releases are only a few times a year, and I'd wanted to use a particular itinerary field added to trip plan responses before 2.5 was released). But admittedly this case didn't have such foresight (I took the latest commit at the time).

Since I already made the corresponding changes in Dotcom to account for the breaking change in the GraphQL API query fields, I'm inclined to leave this commit at this... but in the future, barring feature requirements, I'll likely stick to major version tags to so that we can tackle a bunch of breaking changes more infrequently 👀